app/Models/Product.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    use HasFactory;
+
+    protected $table='products';
+
+    protected $fillable=[
+        'user_id',
+        'product_title',
+        'product_description',
+        'product_price',
+        'image_path',
+        'slug'
+    ];
```

Comment:
add category_id...one category one product

if want to associate many...use tags